edgexfoundry/app-service-configurable
# refactor!: Renamed SecretName to SecretValueKey and SecretnPath to SecretName

BREAKING CHANGE: Renamed SecretName to SecretValueKey and SecretPath to SecretName







**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/app-service-configurable/blob/main/.github/CONTRIBUTING.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions
`make test`
startup edgex
run each profile individually to verify each started up successfully and are functioning correctly
run `./app-service-configurable -p sample -cp`
run `./app-service-configurable -p mqtt-export -cp`
run `./app-service-configurable -p http-export -cp`
run `./app-service-configurable -p functional-tests -cp`
run `./app-service-configurable -p metrics-influxdb -cp`
run `./app-service-configurable -p rules-engine -cp`

## New Dependency Instructions (If applicable)
